src/coreclr/src/dlls/mscoree/unixinterface.cpp

```diff
@@ -137,7 +137,7 @@ static void ConvertConfigPropertiesToUnicode(
         {
             // If this application is a single-file bundle, the bundle-probe callback 
             // is passed in as the value of "BUNDLE_PROBE" property (encoded as a string).
-            *bundleProbe = (BundleProbe*)_wcstoui64(propertyValuesW[propertyIndex], nullptr, 0);
```

Comment:
I don't understand this change. This is just converting the text of the property to number, why does it have to be different? It seems to me that this change actually have any effect - instead of calling wide string to number conversion method on the property converted to wide chars, you call a 8 bit char method on the unconverted 8 bit char version of the property value.

Reply:
Ah true, I was first trying to remove dependency on widechar from BundleProbe all together in coreclr, but it is not required to fix the issue.